doc/classes/DampedSpringJoint2D.xml

```diff
@@ -78,12 +78,16 @@
 	</methods>
 	<members>
 		<member name="damping" type="float" setter="set_damping" getter="get_damping">
+			The damping ratio of the spring joint. A value of 0 indicates an undamped spring, while 1 causes the system to reach equilibrium as fast as possible (critical damping).
 		</member>
 		<member name="length" type="float" setter="set_length" getter="get_length">
+			The maximum length of the spring joint.
 		</member>
 		<member name="rest_length" type="float" setter="set_rest_length" getter="get_rest_length">
+			The resting length of the spring joint. The joint will always try to go to back this length when pulled apart.
```

Comment:
The joint's resting length. 

Does the joint not return to this length if it's compressed?